datadog/resource_datadog_metric_tag_configuration.go

```diff
@@ -37,6 +39,9 @@ func resourceDatadogMetricTagConfiguration() *schema.Resource {
 			if includePercentilesOk && *metricTypeValidated != datadogV2.METRICTAGCONFIGURATIONMETRICTYPES_DISTRIBUTION {
 				return fmt.Errorf("cannot use include_percentiles with a metric_type of %s, must use metric_type of 'distribution'", metricType)
 			}
+			if excludeTagsMode.(bool) && len(tags.(*schema.Set).List()) == 0 {
```

Comment:
I think for this we might want to use an `ok` like we have for the other params to check if the user provided the field or not. This case will not error if the field is set to false and they do not provide tags (I think we want to error for that case right?)

Reply:
I am a bit confused -- I am trying to test that value of the field being `true` (not the presence of a value in the field) with an empty set should return an error. you can have a validation configuration with `{ exclude_tags_mode: false, tags: [] }` which this shouldn't error on

Reply:
ah sorry I thought we were checking to make sure that when the exclude tags field was provided we also had a tags field too. but if we are testing for the empty list case then this makes sense.